scala3doc/src/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -333,9 +334,33 @@ trait ClassLikeSupport:
       else method.paramss
     val genericTypes = if (methodSymbol.isClassConstructor) Nil else method.typeParams
 
+    case class MemberInfo(genericTypes: Map[String, TypeBounds], paramLists: List[Map[String, TypeRepr]], res: TypeRepr)
+
+    def unwrapMemberInfo(c: ClassDef, methodSymbol: Symbol): MemberInfo =
+      val baseTypeRepr = typeForClass(c).asInstanceOf[dotty.tools.dotc.core.Types.Type]
+        .memberInfo(methodSymbol.asInstanceOf[dotty.tools.dotc.core.Symbols.Symbol])(using qctx.asInstanceOf[scala.quoted.runtime.impl.QuotesImpl].ctx)
```

Comment:
it should be moved to `SyntheticsSupport.scala` since we cast to compiler internals?

Reply:
Ok, will fix